main.go

```diff
@@ -65,7 +67,17 @@ var rootCmd = &cobra.Command{
 
 		log.Printf("Starting %s %s", name, version)
 
-		gatewaySrv, err := makeGatewayHandler(saturnOrchestrator, saturnLogger, kuboRPC, gatewayPort, blockCacheSize)
+		var bs blockstore.Blockstore
+		if proxy := os.Getenv(EnvProxyGateway); proxy != "" {
+			bs = newProxyBlockStore(proxy, nil)
+		} else {
+			bs, err = newCabooseBlockStore(saturnOrchestrator, saturnLogger)
```

Comment:
:thought_balloon:  Are we able to flip this around, and use saturn backend only if  `STRN_ORCHESTRATOR_URL` is set, and have default `PROXY_GATEWAY_URLS` set to `http://127.0.0.1:8080` ? 

This way project becomes more useful beyond our infra. One can just run binary locally and it will work, as long they have local Kubo instance.

(change default `--gateway-port` to 8081)